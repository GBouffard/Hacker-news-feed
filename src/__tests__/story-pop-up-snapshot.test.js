import React from 'react';
import StoryPopUp from '../components/story-pop-up';
import renderer from 'react-test-renderer';

describe('StoryPopUp ', () => {
  const component = renderer.create(
    <StoryPopUp
      id={123456789}
      title='this is a snapshot test'
      url='https://github.com/GBouffard/Hacker-news-feed' />
  );

  const tree = component.toJSON();

  it('renders correctly', () => {
    expect(tree).toMatchSnapshot();
  });
});