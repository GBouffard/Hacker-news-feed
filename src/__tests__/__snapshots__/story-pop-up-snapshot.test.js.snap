// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`StoryPopUp  renders correctly 1`] = `
<div
  className="popup"
  id="popup-123456789"
>
  <div
    className="popup__content"
  >
    <div
      className="popup__story-title"
    >
      this is a snapshot test
    </div>
    <a
      className="popup__close-button"
      href="#"
    >
      Ã—
    </a>
    <div
      className="popup__story-text"
    >
      url: 
      https://github.com/GBouffard/Hacker-news-feed
    </div>
  </div>
</div>
`;
